nasa/earthdata-search
# EDSC-4170: Add Admin Preferences Metrics to Admin page

# Overview

### What is the feature?

Add new Preferences Metrics to the admin page.

### What is the Solution?

Created new handler to pull the site_preferences data and calculate the percentage of different values users use as their preferences . New page to display that data.

### What areas of the application does this impact?

List impacted areas.
Admin pages

# Testing

### Reproduction steps

- **Environment for testing:*N/A*
- **Collection to test with:*Any*

1. go to localhost:8080/admin
2. Observe the displayed data matches the users.site_preferences table in the postgresql

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
